### PR TITLE
NEXUS-7883 - Role ordering now case sensitive

### PR DESCRIPTION
**Pull request by kellyrob99**
- update the ExtJS model with an appropriate sorter and enhance existing functional test to confirm behaviour

### CODE REVIEW

File: plugins/rapture/nexus-coreui-plugin/src/main/resources/static/rapture/NX/coreui/model/Role.js [hunk 1]
```diff
@@ -23,7 +23,7 @@ Ext.define('NX.coreui.model.Role', {
     'id',
     'version',
     'source',
-    'name',
+    {name: 'name', sortType: 'asUCText'},
```
Comment (jdillon) [inline plugins/rapture/nexus-coreui-plugin/src/main/resources/static/rapture/NX/coreui/model/Role.js]: minor style note, may be nicer to uniformlly use object config and not mix, /me shrugs

**Comment by jdillon** [comment]
+1 with minor comment

File: plugins/rapture/nexus-coreui-plugin/src/main/resources/static/rapture/NX/coreui/model/Role.js (hunk 1 above)
Comment (kellyrob99) [inline plugins/rapture/nexus-coreui-plugin/src/main/resources/static/rapture/NX/coreui/model/Role.js]: 01dd9de9cd00ee427f9a7850cbd46d9c6a599ca7

**Comment by djsauble** [comment]
+1

**Pull request merged**